setup.py

```diff
@@ -82,6 +82,9 @@
         # transformers 4.10.0 will break the translation model we used here
         "nlp": ["texar-pytorch>=0.1.4"],
         "extractor": ["texar-pytorch>=0.1.4"],
+        "payload": ["soundfile>=0.10.3",
```

Comment:
This update should be reflected in [README](https://github.com/asyml/forte#installation). And maybe we can also remove [audio_ext](https://github.com/asyml/forte/pull/859/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R64)?

Reply:
added in readme.md. 
By remove `audio_ext`, do you mean also rewrite the example by using the new payload design?